pa11y/mappings/common/createCommonMappings.ts

```diff
@@ -0,0 +1,90 @@
+import { Options } from "wiremock-rest-client/dist/model/options.model";
+import wiremock from "../../lib/general";
+
+export default (baseUri?: string, options?: Options) => {
+  // Mapping for covers.
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "GET",
+      urlPattern: "/api/v2/covers.*",
+    },
+    response: {
+      jsonBody: [
+        {
+          id: "870970-basis:134693959",
```

Comment:
Consider if it is relevant to template the id in the response from the request.

This always returns a fixed id.

Reply:
👍